### PR TITLE
rf#113  Move some ecommerce description to SE side

### PR DESCRIPTION
**Pull request by duker33**
Related to fidals/refarm-site#113

### CODE REVIEW

File: ecommerce.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+Сайт берёт товары только из 1С, не из админки.
+И имеет специфичный алгоритм расчёта скидки.
+
+## Алгоритм
+
+1C формирует выгрузку базы товаров в формате .xml и помещает её в определённой директории на Windows Server’е (контролируемым бизнесом). Выгрузка скачивается по ftp.
+
+*Примечание: в случае проблем с ftp или Windows Server’ом обращайтесь к Хелхи.*
+
+Чтобы выгрузка из 1С оказалась в своей базе, запустите на сервере команду "dcp exec se-python python manage.py update_db".
```
Comment (artemiy312) [inline ecommerce.md]: "dcp exec ~~se-python~~ app python manage.py ~~update_db~~ update_catalog"

File: ecommerce.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+*Примечания:*
+1. Команда "update_db" запускается по крону (crone) раз в 2 часа.
```
Comment (artemiy312) [inline ecommerce.md]: update_db -> update_catalog

File: ecommerce.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+*Пример 1*
+
+*Пользователь указал количество: 500 шт. Расчёт по розничной цене: 500 х 1111 = 555500 руб. 555500 > 100000, поэтому расчёт идёт по цене 666,16 руб за единицу. Итоговая цена: 500 х 666,16 = 333080 руб.*
+
+*Пример 2*
+
+*Возьмём товар - блок питания: https://www.shopelectro.ru/catalog/products/2805/. Закажем 26 шт. Видим в корзине это: 1111 руб х 26 шт. = 28886 руб. Очистим корзину. Закажем 27 шт. Видим в корзине это: 767 руб х 27 шт. = 20717 руб. Получается, мы можем купить 27 одних и тех же блоков дешевле, чем 26.*
+
+![](img/01.png)
```
Comment (artemiy312) [inline ecommerce.md]: default

Seems github does not render anything by this. It is not redundant links?

File: ecommerce.md (hunk 1 above)
Comment (duker33) [reply ecommerce.md]: TODO

File: ecommerce.md (hunk 2 above)
Comment (duker33) [reply ecommerce.md]: TODO

File: ecommerce.md (hunk 3 above)
Comment (duker33) [reply ecommerce.md]: it's wrong markdown, i suppose.

TODO

**Pull request merged**